src/Settings.php

```diff
@@ -4,229 +4,233 @@
 
 use Tribe__Settings_Manager;
 
-/**
- * Do the Settings.
- */
-class Settings {
-
-	/**
-	 * The Settings Helper class.
-	 *
-	 * @var Settings_Helper
-	 */
-	protected $settings_helper;
-
-	/**
-	 * The prefix for our settings keys.
-	 *
-	 * Gets set automatically from the Text Domain or can be set manually.
-	 * The prefix should not end with underscore `_`.
-	 *
-	 * @var string
-	 */
-	private $opts_prefix = '';
-
+if ( ! class_exists( Settings::class ) ) {
 	/**
-	 * Settings constructor.
-	 *
-	 * TODO: Update this entire class for your needs, or remove the entire `src` directory this file is in and do not load it in the main plugin file.
+	 * Do the Settings.
 	 */
-	public function __construct( $opts_prefix = '' ) {
-		$this->settings_helper = new Settings_Helper();
-
-		$this->set_options_prefix( $opts_prefix );
-
-		// Remove settings specific to Google Maps
-		add_action( 'admin_init', [ $this, 'remove_settings' ] );
-
-		// Add settings specific to OSM
-		add_action( 'admin_init', [ $this, 'add_settings' ] );
-	}
-
-	/**
-	 * Set the options prefix to be used for this extension's settings.
-	 *
-	 * Defaults to the text domain, converting hyphens to underscores.
-	 * Always has ends with a single underscore.
-	 *
-	 * @param string $opts_prefix
-	 */
-	private function set_options_prefix( $opts_prefix = '' ) {
-		if ( empty( $opts_prefix ) ) {
-			$opts_prefix = str_replace( '-', '_', PLUGIN_TEXT_DOMAIN );
+	class Settings {
+
+		/**
+		 * The Settings Helper class.
+		 *
+		 * @var Settings_Helper
+		 */
+		protected $settings_helper;
+
+		/**
+		 * The prefix for our settings keys.
+		 *
+		 * Gets set automatically from the Text Domain or can be set manually.
+		 * The prefix should not end with underscore `_`.
+		 *
+		 * @var string
+		 */
+		private $opts_prefix = '';
+
+		/**
+		 * Settings constructor.
+		 *
+		 * TODO: Update this entire class for your needs, or remove the entire `src` directory this file is in and do not load it in the main plugin file.
+		 *
+		 * @param string $opts_prefix
+		 */
+		public function __construct( $opts_prefix = '' ) {
+			$this->settings_helper = new Settings_Helper();
+
+			$this->set_options_prefix( $opts_prefix );
+
+			// Remove settings specific to Google Maps
+			add_action( 'admin_init', [ $this, 'remove_settings' ] );
+
+			// Add settings specific to OSM
+			add_action( 'admin_init', [ $this, 'add_settings' ] );
 		}
 
-		$opts_prefix = $opts_prefix . '_';
+		/**
+		 * Set the options prefix to be used for this extension's settings.
+		 *
+		 * Defaults to the text domain, converting hyphens to underscores.
+		 * Always has ends with a single underscore.
+		 *
+		 * @param string $opts_prefix
+		 */
+		private function set_options_prefix( $opts_prefix = '' ) {
+			if ( empty( $opts_prefix ) ) {
+				$opts_prefix = str_replace( '-', '_', PLUGIN_TEXT_DOMAIN );
```

Comment:
Lets avoid this being set like this, we should use something that would error out, so someone couldn't copy and pasta this without the replacement.

Reply:
done